Build tags failing for Raspberry Pi.

Build tags do not seem to take into account for the Raspberry Pi. It runs on an ARM Cortex-A72 chip. Is this intended?

```shell
github.com/oasislabs/ed25519/internal/curve25519
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:35:32: undefined: Bignum25519
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:37:12: undefined: Bignum25519
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:40:2: undefined: SquareTimes
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:41:2: undefined: Mul
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:42:2: undefined: SquareTimes
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:43:2: undefined: Mul
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:44:2: undefined: SquareTimes
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:45:2: undefined: Mul
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:57:20: undefined: Bignum25519
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:73:31: undefined: Bignum25519
../../go/pkg/mod/github.com/oasislabs/ed25519@v0.0.0-20191122104632-9d9ffc15f526/internal/curve25519/helpers.go:45:2: too many errors
```

--- a/internal/curve25519/curve25519_donna_32bit.go
+++ b/internal/curve25519/curve25519_donna_32bit.go
@@ -26,10 +26,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// +build !amd64
-// +build !go1.13,arm64
-// +build !go1.13,ppc64le
-// +build !go1.13,ppc64
+// +build 386 arm !go1.13,arm64 !go1.13,ppc64le !go1.13,ppc64 mips mipsle mips64le mips64 s390x
 
 package curve25519
 

--- a/internal/ge25519/tables_32bit.go
+++ b/internal/ge25519/tables_32bit.go
@@ -26,10 +26,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// +build !amd64
-// +build !go1.13,arm64
-// +build !go1.13,ppc64le
-// +build !go1.13,ppc64
+// +build 386 arm !go1.13,arm64 !go1.13,ppc64le !go1.13,ppc64 mips mipsle mips64le mips64 s390x
 
 package ge25519
 

--- a/internal/modm/modm_32bit.go
+++ b/internal/modm/modm_32bit.go
@@ -26,10 +26,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// +build !amd64
-// +build !go1.13,arm64
-// +build !go1.13,ppc64le
-// +build !go1.13,ppc64
+// +build 386 arm !go1.13,arm64 !go1.13,ppc64le !go1.13,ppc64 mips mipsle mips64le mips64 s390x
 
 package modm
 

